build_tools/docker/bazel/gcr_update.sh

```diff
@@ -21,18 +21,19 @@ set -e
 # Ensure correct authorization.
 gcloud auth configure-docker
 
+# Determine which image tag to update. Updates :latest by default.
+if [[ $* == --update-prod ]]; then
```

Comment:
Nothing particularly about flags, but this incorporates some of their other guidelines:
```bash
# Determine which image tag to update. Updates :latest by default.
TAG="latest"
if [[ "$#" -ne 0  ]]; then
  if [[ "$@" == "--update-prod" ]]; then
    TAG="prod"
  else
    echo "Invalid commandline arguments. Accepts --update-prod or no arguments."
    exit 1
  fi
fi
echo "Updating $TAG"
```
Thoughts?

Reply:
Yeah any actual program is going to be written in a real language :-D So of course inside Google there's a nice way to do this with a library for actual flag support, but it's not open sourced. Based on some searching, I think the typical bash way is with getopt[s]. I'm ok with what you wrote here though, since you're only trying to parse one flag. Of course it will totally break down if you ever try to add more

Reply:
Sounds good. And of course lol. Ideally this should never need to get any more complicated, but if it does it would be easy to iterate through all of the command line arguments and use a case statement to parse the flags.